pom.xml

```diff
@@ -103,43 +103,43 @@
-        <version.org.awaitility>3.1.0</version.org.awaitility>
+        <version.org.awaitility>3.1.5</version.org.awaitility>
         <version.org.codehaus.mojo.keytool-maven-plugin>1.5</version.org.codehaus.mojo.keytool-maven-plugin>
-        <version.org.eclipse.microprofile>1.2</version.org.eclipse.microprofile>
-        <version.org.glassfish.tyrus>1.13.1</version.org.glassfish.tyrus>
+        <version.org.eclipse.microprofile>1.4</version.org.eclipse.microprofile>
+        <version.org.glassfish.tyrus>1.14</version.org.glassfish.tyrus>
         <version.org.honton.chas.process-exec-maven-plugin>0.9.2</version.org.honton.chas.process-exec-maven-plugin>
-        <version.org.infinispan>9.3.1.Final</version.org.infinispan>
-        <version.org.jboss.arquillian>1.2.1.Final</version.org.jboss.arquillian> <!-- some tests fail with 1.3+, not yet sure why -->
+        <version.org.infinispan>9.4.3.Final</version.org.infinispan>
+        <version.org.jboss.arquillian>1.4.0.Final</version.org.jboss.arquillian>
         <version.org.jboss.arquillian.drone>2.5.1</version.org.jboss.arquillian.drone>
         <version.org.jboss.remotingjmx>3.0.0.Final</version.org.jboss.remotingjmx>
-        <version.org.jboss.resteasy>3.6.1.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>3.6.2.Final</version.org.jboss.resteasy>
         <version.org.jboss.spec.javax.ejb.jboss-ejb-api_3.2_spec>1.0.0.Final</version.org.jboss.spec.javax.ejb.jboss-ejb-api_3.2_spec>
-        <version.org.jboss.spec.javax.json.jboss-json-api_1.0_spec>1.0.0.Final</version.org.jboss.spec.javax.json.jboss-json-api_1.0_spec>
-        <version.org.jboss.spec.javax.servlet.jboss-servlet-api_3.0_spec>1.0.2.Final</version.org.jboss.spec.javax.servlet.jboss-servlet-api_3.0_spec>
-        <version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.0_spec>1.0.0.Final</version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.0_spec>
-        <version.org.jboss.spec.web.7.0.api>1.1.0.Final</version.org.jboss.spec.web.7.0.api>
-        <version.org.jboss.spec.all.7.0.api>1.1.0.Final</version.org.jboss.spec.all.7.0.api>
-        <version.org.jgroups>4.0.13.Final</version.org.jgroups>
-        <version.org.keycloak.everything>4.7.0.Final</version.org.keycloak.everything>
+        <version.org.jboss.spec.javax.json.jboss-json-api_1.1_spec>1.0.0.Final</version.org.jboss.spec.javax.json.jboss-json-api_1.1_spec>
+        <version.org.jboss.spec.javax.servlet.jboss-servlet-api_4.0_spec>1.0.0.Final</version.org.jboss.spec.javax.servlet.jboss-servlet-api_4.0_spec>
+        <version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.1_spec>1.0.2.Final</version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.1_spec>
+        <version.org.jboss.spec.web.8.0.api>1.0.2.Final</version.org.jboss.spec.web.8.0.api>
+        <version.org.jboss.spec.all.8.0.api>1.0.2.Final</version.org.jboss.spec.all.8.0.api>
```

Comment:
I thought on doing this, but, as these tests doesn't (still) use Java EE 8 I didn't consider that necessary. But it's good to keep everything to date for sure.